use-timescale/migration/migrate-hypershift.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Migrate to Timescale Cloud with Hypershift
+excerpt: Migrate an existing PostgreSQL database to Timescale Cloud using Hypershift
+products: [cloud]
+keywords: [data migration, Hypershift]
+tags: [ingest, Hypershift, postgresql]
+---
+
+import HypershiftIntro from "versionContent/_partials/_hypershift-intro.mdx";
+import CloudTrial from "versionContent/_partials/_cloudtrial.mdx";
+
+# Migrate with Hypershift
+
+<HypershiftIntro />
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+*   Signed up for your [free Timescale Cloud account][cloud-install].
+*   Installed [Docker][docker-install].
+
+<CloudTrial />
+
+It is recommended that you create an index on the `time` column of tables in your source
+database before you begin the migration. Hypershift works without an index,
```

Comment:
This is only relevant for tables which are to be converted into hypertables. The way this is phrased, it sounds like this is a precondition for _all_ tables, which it is not.

Reply:
```suggestion
Before you begin the migration, it is recommended that you create an index on the `time` column of tables in your source database that are going to be converted to hypertables. Hypershift works without an index,
```